pkg/leaderelection/leaderelection.go

```diff
@@ -72,9 +74,9 @@ func New(log logr.Logger, name, namespace string, kubeClient kubernetes.Interfac
 	e.leaderElectionCfg = leaderelection.LeaderElectionConfig{
 		Lock:            e.lock,
 		ReleaseOnCancel: true,
-		LeaseDuration:   15 * time.Second,
-		RenewDeadline:   10 * time.Second,
-		RetryPeriod:     2 * time.Second,
+		LeaseDuration:   6 * retryPeriod,
+		RenewDeadline:   5 * retryPeriod,
```

Comment:
isn't it to aggressive

Reply:
Why ? Will this be more aggressive than before ?

Reply:
We are reducing  `LeaseDuration` time to 6 sec which too less IMO,  if we have verified this behaviour i am good with changes.

Reply:
We are not reducing to `6s` but `6 * retryPeriod` (`retryPeriod = 2s` by default).

Reply:
aah my bad, looks good then.